### PR TITLE
[explore-v2] make chart container work with existing visualization files

### PR DESCRIPTION
**Pull request by ascott**
- makes `<ChartContainer />` use existing visualization js files to render charts.

todo:
- [x] use vizMap object to map chart type to vis module
  https://github.com/ascott/caravel/blob/master/caravel/assets/visualizations/main.js#L3
- [x] figure out if other vis modules have other different dependencies than than the `nvd3_vis` module
  ---- added an issue documenting all the issues with wrapping vis's in V2. will come back for these in a follow up PR. https://github.com/airbnb/caravel/issues/1341

/exploreV2/ with line, bar, area, pie, bubble, box_plot

screenshot 2016-10-12 18 15 36
screenshot 2016-10-12 18 15 57
screenshot 2016-10-12 18 16 21
screenshot 2016-10-12 18 17 06
screenshot 2016-10-12 18 17 34
screenshot 2016-10-12 18 18 08

cc @mistercrunch @vera-liu

### CODE REVIEW

File: caravel/assets/javascripts/explorev2/components/ChartContainer.jsx [hunk 1]
```diff
@@ -3,28 +3,63 @@ import { connect } from 'react-redux';
 import { Panel } from 'react-bootstrap';
 import TimeSeriesLineChart from './charts/TimeSeriesLineChart';
 import moment from 'moment';
+import nvd3Vis from '../../../visualizations/nvd3_vis';
```
Comment (mistercrunch) [inline caravel/assets/javascripts/explorev2/components/ChartContainer.jsx]: To generalize here you'll be able to import the object that maps the visualization type string to the object, it's in `../../../visualizations/nvd3_vis`

File: caravel/assets/javascripts/explorev2/components/ChartContainer.jsx [hunk 2]
```diff
@@ -3,28 +3,63 @@ import { connect } from 'react-redux';
...
+      },
+    };
   }
 
-  isLineViz() {
-    // todo(alanna) generalize this check and map to charts
-    return this.props.vizType === 'line';
+  renderVis() {
+    const slice = this.getMockedSliceObject();
+    nvd3Vis(slice).render();
```
Comment (mistercrunch) [inline caravel/assets/javascripts/explorev2/components/ChartContainer.jsx]: then here use: `vizMap[vizType](slice).render()`

**Comment by ascott** [comment]
table type has a dependency on slice.d3format:

screenshot 2016-10-13 14 36 06

**Comment by ascott** [comment]
big number with trendline doesn't show big number

screenshot 2016-10-13 14 41 53

**Comment by ascott** [comment]
big number doesn't render big number:

screenshot 2016-10-13 14 42 55
screenshot 2016-10-13 14 43 07

**Comment by ascott** [comment]
directed force layout looks a little different.... 

screenshot 2016-10-13 14 44 39
screenshot 2016-10-13 14 44 54

**Comment by ascott** [comment]
region filter is broken on /explore/

screenshot 2016-10-13 14 46 20

/exploerV2/ has dependency on `slice.getFilters`
screenshot 2016-10-13 14 47 07

**Comment by ascott** [comment]
heat map has some layout issues:
screenshot 2016-10-13 14 48 12
screenshot 2016-10-13 14 48 28

**Comment by ascott** [comment]
pivot table type throws an error in V2:

screenshot 2016-10-13 14 52 57
screenshot 2016-10-13 14 53 24

**Comment by ascott** [comment]
sankey as layout/visual issues in V2:

screenshot 2016-10-13 14 54 17
screenshot 2016-10-13 14 54 36

**Comment by ascott** [comment]
sunburst has layout/visual issues in V2:
screenshot 2016-10-13 14 55 16
screenshot 2016-10-13 14 55 59

**Comment by ascott** [comment]
word_map type throws an error in V2:

screenshot 2016-10-13 14 58 35
screenshot 2016-10-13 14 58 49

**Comment by ascott** [comment]
@mistercrunch added support to use the mapping from vis type to vis file. i removed some components I had duplicated in explorev2, and also removed the new line chart in favor of using the existing viz files. i added an issue documenting the outstanding issues with wrapping the vis's in V2 and will make another PR with fixes for those. https://github.com/airbnb/caravel/issues/1341. PTAL!

**Pull request merged**